contracts/src/v0.8/ccip/test/applications/PingPongDemo.t.sol

```diff
@@ -30,75 +30,23 @@ contract PingPong_startPingPong is PingPongDappSetup {
   uint256 internal pingPongNumber = 1;
 
   function test_StartPingPong_With_Sequenced_Ordered_Success() public {
-    Client.EVM2AnyMessage memory sentMessage = Client.EVM2AnyMessage({
-      receiver: abi.encode(i_pongContract),
-      data: abi.encode(pingPongNumber),
-      tokenAmounts: new Client.EVMTokenAmount[](0),
-      feeToken: s_sourceFeeToken,
-      extraArgs: Client._argsToBytes(Client.EVMExtraArgsV1({gasLimit: 200_000}))
-    });
-
-    uint256 expectedFee = s_sourceRouter.getFee(DEST_CHAIN_SELECTOR, sentMessage);
-
-    Internal.EVM2EVMMessage memory message = Internal.EVM2EVMMessage({
-      sequenceNumber: 1,
-      feeTokenAmount: expectedFee,
-      sourceChainSelector: SOURCE_CHAIN_SELECTOR,
-      sender: address(s_pingPong),
-      receiver: i_pongContract,
-      nonce: 1,
-      data: abi.encode(pingPongNumber),
-      tokenAmounts: sentMessage.tokenAmounts,
-      sourceTokenData: new bytes[](sentMessage.tokenAmounts.length),
-      gasLimit: 200_000,
-      feeToken: sentMessage.feeToken,
-      strict: false,
-      messageId: ""
-    });
-
-    _assertPingPongSuccess(message);
+    _assertPingPongSuccess();
   }
 
   function test_StartPingPong_With_OOO_Success() public {
     s_pingPong.setOutOfOrderExecution(true);
 
-    Client.EVM2AnyMessage memory sentMessage = Client.EVM2AnyMessage({
-      receiver: abi.encode(i_pongContract),
-      data: abi.encode(pingPongNumber),
-      tokenAmounts: new Client.EVMTokenAmount[](0),
-      feeToken: s_sourceFeeToken,
-      extraArgs: Client._argsToBytes(Client.EVMExtraArgsV2({gasLimit: 200_000, allowOutOfOrderExecution: true}))
-    });
-
-    uint256 expectedFee = s_sourceRouter.getFee(DEST_CHAIN_SELECTOR, sentMessage);
-
-    Internal.EVM2EVMMessage memory message = Internal.EVM2EVMMessage({
-      sequenceNumber: 1,
-      feeTokenAmount: expectedFee,
-      sourceChainSelector: SOURCE_CHAIN_SELECTOR,
-      sender: address(s_pingPong),
-      receiver: i_pongContract,
-      nonce: 0,
-      data: abi.encode(pingPongNumber),
-      tokenAmounts: sentMessage.tokenAmounts,
-      sourceTokenData: new bytes[](sentMessage.tokenAmounts.length),
-      gasLimit: 200_000,
-      feeToken: sentMessage.feeToken,
-      strict: false,
-      messageId: ""
-    });
-
-    _assertPingPongSuccess(message);
+    _assertPingPongSuccess();
   }
 
-  function _assertPingPongSuccess(Internal.EVM2EVMMessage memory message) internal {
-    message.messageId = Internal._hash(message, s_metadataHash);
-
+  function _assertPingPongSuccess() internal {
     vm.expectEmit();
     emit PingPongDemo.Ping(pingPongNumber);
 
-    vm.expectEmit();
-    emit EVM2EVMOnRamp.CCIPSendRequested(message);
+    Internal.EVM2AnyRampMessage memory message;
+
+    vm.expectEmit(false, false, false, false);
```

Comment:
We don't care about the event data, just that it was emitted. This removed all of the version specific code above